static/app/components/profiling/flamegraph/continuousFlamegraph.tsx

```diff
@@ -219,6 +250,28 @@ export function ContinuousFlamegraph(): ReactElement {
     return profileGroup.profiles.find(p => p.threadId === flamegraphProfiles.threadId);
   }, [profileGroup, flamegraphProfiles.threadId]);
 
+  const spanTree: SpanTree = useMemo(() => {
+    if (segment.type === 'resolved' && segment.data) {
+      return new SpanTree(
+        segment.data,
+        collectAllSpanEntriesFromTransaction(segment.data)
+      );
+    }
+
+    return LOADING_OR_FALLBACK_SPAN_TREE;
+  }, [segment]);
```

Comment:
We talked about this but it'd be great if we could pick a different name other than `segment` for hits.

Reply:
Yeah, I went ahead and renamed it to eventId